lang/ar.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} غير صالحة.",
 	"components.form-element.defaultFieldLabel": "الحقل",
 	"components.form-element.input.email.typeMismatch": "البريد الإلكتروني غير صالح",
-	"components.form-element.input.number.rangeError": "يجب أن يكون الرقم {minExclusive, select, true {أكبر من} false {أكبر من أو مساويًا لـ}} {min} و{maxExclusive, select, true {أقل من} false {أقل من أو مساويًا لـ}} {max}.",
-	"components.form-element.input.number.rangeOverflow": "يجب أن يكون الرقم {maxExclusive, select, true {أقل من} false {أقل من أو مساويًا لـ}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "يجب أن يكون الرقم {minExclusive, select, true {أكبر من} false {أكبر من أو مساويًا لـ}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Number must be less than {max}.} other {Number must be less than or equal to {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Number must be greater than {min}.} other {Number must be greater than or equal to {min}.}}",
```

Comment:
Just reset all the Arabic ones as all my attempts to do this automatically with RTL pasted stuff in the wrong place.